### PR TITLE
fixed some spanish translations

### PR DESCRIPTION
**Pull request by arp43b**
## Description
A few sentences describing the overall goals of the pull request' s commits.

## Issues
 - Fix #<issue>
 - ...

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [ ] Yes, and I documented them in migration notes
 - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: web/client/translations/data.es-ES [hunk 1]
```diff
@@ -871,17 +846,16 @@
...
             "creatingUser": "Creando...",
-            "userCreated": "!Creado!",
+            "userCreated": "¡Creado!",
             "deleting": "Borrando...",
             "delete": "Borrar",
             "confirmDeleteUser": "¿Está seguro de borrar el usuario?",
-            "errorDelete": "A ocurrido un error al borrar el usuario:",
-            "errorSaving": "A ocurrido un error al guardar el usuario:",
-            "selectedGroups": "GRUPOS SELECCIONADOS",
-            "requiredFiedsMessage": "Se requiere llenar campos marcados con asterisco (*)"
```
Comment (MV88) [inline web/client/translations/data.es-ES]: hi, please do not remove any record

File: web/client/translations/data.es-ES [hunk 2]
```diff
@@ -903,51 +877,47 @@
...
             "forceDrawer": "Mostrar el contenido",
-            "apiLinkTitle": "Via APIs",
-            "QRCodeLinkTitle": "código qr",
+            "apiLinkTitle": "A través de APIs",
+            "QRCodeLinkTitle": "Código QR",
             "msgCopiedUrl": "Url Copiada",
             "msgToCopyUrl": "Pinche para copiar",
-            "sharedTitle": "Mira mi nuevo mapa: ",
-            "advancedOptions": "Opciones avanzadas",
-            "addBboxParam": "Añadir bbox param al enlace de compartir",
```
Comment (MV88) [inline web/client/translations/data.es-ES]: same as above, please do not remove any record

**Comment by mbarto** [comment]
@MV88, @arp43b I should have added back all the missing stuff, please check

**Comment by mbarto** [comment]
Ok, it looks good now.
Thanks for the contribution @arp43b much appreciated!

**Pull request merged**